OptimoveSDK/Sources/Classes/Optimobile/Optimobile+Location.swift

```diff
@@ -0,0 +1,46 @@
+
+
+import Foundation
+import CoreLocation
+
+extension Optimobile {
+    static func sendLocationUpdate(location: CLLocation) {
+        let parameters = [
+            "lat" : location.coordinate.latitude,
+            "lng" : location.coordinate.longitude
+        ]
+
+        Optimobile.trackEvent(eventType: OptimobileEvent.ENGAGE_LOCATION_UPDATED, properties: parameters, immediateFlush: true)
+    }
+    
+    static func trackIBeaconProximity(beacon: CLBeacon) {
+        var parameters = [
+            "type": 1,
+            "major": beacon.major.intValue,
```

Comment:
in the old sdk we had them as strings for some reason https://github.com/Kumulos/KumulosSdkSwift/blob/master/Sources/SDK/Kumulos%2BEngage.swift#L27

but 
1. spec really has them as 16 bit ints
2. push-service will handle ints fine, they are only used in one `sprintf`